Bandwidth.Standard/Voice/Bxml/Bxml.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+
+namespace Bandwidth.Standard.Voice.Bxml
+{
+  /// <summary>
+  ///   Bxml class for Bandwidth XML
+  /// </summary>
+  [XmlRoot("Bxml", Namespace = "")]
+  public class BXML : IXmlSerializable
+  {
+    private static readonly XmlSerializer Serializer = new XmlSerializer(typeof (BXML), "");
+    private readonly List<IVerb> _list = new List<IVerb>();
+
+        private static readonly Regex XML_REGEX = new Regex("&lt;([a-zA-Z//].*?)&gt;");
+
+        private static readonly Regex SPEAK_SENTENCE_REGEX = new Regex("<SpeakSentence.*?>.*?<\\/SpeakSentence>");
+
+        /// <summary>
+        ///   Default constructor
+        /// </summary>
+        public BXML()
+    {
+    }
+
+    /// <summary>
+    ///   Constructor with verbs
+    /// </summary>
+    /// <param name="verbs">verbs to be added to Bxml</param>
+    public BXML(params IVerb[] verbs)
+    {
+      _list.AddRange(verbs);
+    }
+
+    XmlSchema IXmlSerializable.GetSchema() => null;
+
+    void IXmlSerializable.ReadXml(XmlReader reader)
+    {
+      throw new NotImplementedException();
```

Comment:
This throws an exception when called. This should be implemented before we release.

Reply:
@hamermike this comes from Response.cs - can we just remove it? Not even sure what this method is supposed to do